capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -0,0 +1,65 @@
+| Disk Addition to Instance                  | Yes                       | Yes             | No                          | Yes                        | No                                    |
+| Key Pairs Inventory                        | Yes                       | No              | Yes                         | Yes                        | Yes                                   |
+| Key Pairs Management                       | Yes                       | Yes             | No                          | Yes                        | Yes                                   |
+| Reporting                                  | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
+| Right Sizing                               | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Chargeback                                 | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
+| Remote Console Instance Access             | No                        | No              | No                          | Yes                        | Yes (for NovaLink-managed hosts only) |
+| Snapshot Creation and Removal              | No                        | No              | No                          | Yes                        | Yes                                   |
+| Instance Compliance Enforcement            | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
+| Instance Policy Enforcement                | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Instance Power Operations                  | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Instance Retirement                        | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Alerts - Real Time                         | No                        | No              | No                          | Yes                        | No                                    |
+| Alerts - Instance Value Changed            | No                        | No              | No                          | No                         | No                                    |
+| Alerts - Reconfigured                      | No                        | No              | No                          | No                         | No                                    |
```

Comment:
In the infrastructure manager section this is called `Alerts - Vm Reconfigured` https://github.com/ManageIQ/manageiq-documentation/pull/1474/files#diff-4a180079816a0cde7d2cf11432c74a3cR24

Reply:
ok - so changing to Instance Reconfigured?

Reply:
Since these all say not supported I think this should match `Vm Reconfigured`, I assume this is talking about the same concept otherwise them all saying No would mean nothing supports this feature :)

Reply:
I think the cloud names VMs as Instances.  So, Instance Reconfigured is probably more appropriate in this table.